src/IxMilia.Dxf/Entities/DxfLwPolyline.cs

```diff
@@ -105,18 +105,49 @@ internal override DxfEntity PopulateFromBuffer(DxfCodePairBufferReader buffer)
 
         protected override IEnumerable<DxfPoint> GetExtentsPoints()
         {
-            DxfPoint? lastLocation = null;
-            foreach (var vertex in Vertices)
+            int n = Vertices.Count;
+            for (int i = 0; i < n; i++)
             {
-                var current = new DxfPoint(vertex.X, vertex.Y, 0.0);
-                yield return current;
-                if (vertex.Bulge != 0.0 && lastLocation != null)
+                var vertex = Vertices[i];
+                if (Math.Abs(vertex.Bulge) <= 1e-10)
                 {
-                    // TODO: the segment between `lastLocation` and `current` is an arc
+                    yield return new DxfPoint(vertex.X, vertex.Y, 0);
                 }
+                else
+                {
+                    // the segment between `vertex.Location` and `next.Location` is an arc
+                    var next = Vertices[(i + 1) % n];
```

Comment:
This index wrapping magic (`% n`) is only valid if `IsClosed` is true and we're on the last vertex.  If the bulge on the last vertex is zero then that won't matter because the resultant bounding box will be the same, but this branch is only executed if the bulge is significant (see line 112) and on the off chance that `IsClosed` is false then it's possible that the bounding box could be artificially inflated to include an non-extant arc segment.

Reply:
Give me a few minutes, will migrate some code from my project.